modules/alibaba-ai-installing.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
```

Comment:
I recommend that you move these steps from a procedure module to the assembly because they are high-level steps. 

This change would have the advantage of being able to add links and xrefs to each step instead of being in an "Additional resources" section.

Reply:
I tried it like that initially and it looked weird and confusing. I also wanted the overview section to say "Installing on Alibaba Cloud" only to coordinate with the rest of the installation sections.

Reply:
But the problem is that you end up with title repetition because the assembly and module titles are nearly the same.